package.json

```diff
@@ -52,21 +50,19 @@
     "jest-junit": "^13",
     "jsii": "^1.54.0",
     "jsii-diff": "^1.54.0",
-    "jsii-docgen": "^6.1.9",
+    "jsii-docgen": "^6.1.12",
     "json-schema": "^0.4.0",
     "npm-check-updates": "^12",
-    "projen": "^0.52.37",
+    "projen": "^0.52.44",
     "standard-version": "^9",
     "ts-jest": "^27.1.3",
     "typescript": "^4.5.5"
   },
   "peerDependencies": {
-    "@aws-cdk/core": "^1.123.0",
     "constructs": "^3.2.27",
     "monocdk": "^1.123.0"
   },
   "dependencies": {
-    "@aws-cdk/core": "^1.123.0",
     "monocdk": "^1.123.0"
```

Comment:
Need to remove this too, it should only be in peer/dev deps.

Reply:
I do not see it locally. Maybe i updated the PR in a wrong way. Let me check.

Reply:
Is it still there? https://github.com/cdklabs/cdk-monitoring-constructs/blob/4b9118b4104698803c300b6b20925ec07757a59c/package.json

Reply:
Removed it as well.